cmds/exp/netbootxyz/netbootxyz.go

```diff
@@ -175,7 +175,7 @@ func (o OSEndpoint) Load() error {
 			fmt.Printf("With Initrd at %s\n", tmpPath+"initrd")
 			fmt.Printf("Commandline: %s\n", o.Commandline)
 			// Load Kernel and initrd
-			err = kexec.FileLoad(vmlinuz, initrd, o.Commandline)
+			kexec.FileLoad(vmlinuz, initrd, o.Commandline)
 			// Load KExec kernel and initrd - init cmdline
```

Comment:
this code is kind of crazy anyway. It uses the kexec package and ignores the error. That ought to get fixed, not ignored :-)

Reply:
see #2529